tokenserver/tests/assignment/test_sqlnode.py

```diff
@@ -28,13 +28,8 @@ def setUp(self):
         self.backend.add_service('sync-1.0', '{node}/1.0/{uid}')
         self.backend.add_service('sync-1.5', '{node}/1.5/{uid}')
         self.backend.add_service('queuey-1.0', '{node}/{service}/{uid}')
-        self.backend.add_service('spanner',
-                                 'https://spanner.example.com/1.5/{uid}')
         self.backend.add_node('sync-1.0', 'https://phx12', 100)
         self.backend.add_node('sync-1.5', 'https://phx11', 100)
-        self.backend.add_node('spanner',
-                              'https://spanner.example.com/1.5/{uid}', 9000)
-        self.backend._migrate_new_user_percentage = 0
```

Comment:
I don't think these were used for anything. We could consider adding additional tests in this file, e.g. to confirm that percentage-based selection works as we expect when you change the percentage.